src/chapters/forms/select-input.rst

```diff
@@ -0,0 +1,80 @@
+Select Input
+============
+
+Select inputs create a clickable menu that displays options and allows the user
+to select one. The available options are defined by ``<option>`` tags inside of
+the ``<select></select>`` tag.
+
+``<option>`` tags have a ``value`` attribute which defines the value submitted
+if that option is selected. The text inside the
+``<option>Option text</option>`` is what is displayed in the select menu.
+
```

Comment:
I'm wanting a little more information about  setting a Default <option>. See 20.6.1. in lchs book for reference

Reply:
added a little on this topic